src/SDDPoptimize.jl

```diff
@@ -292,6 +285,13 @@ function build_model(model, param, t)
         @objective(m, Min, cost + alpha)
     end
 
+    # Add binary variable if problem is a SMIP:
+    if model.IS_SMIP
+        m.colCat[1:nx] = model.stateCat
```

Comment:
I guess people that might use this library would hopefully know a bit about SDDP... Personally, I would either comment it out of put a nice warning.